InternalTimeSheet/app/src/main/res/layout/activity_dashboard.xml

```diff
@@ -10,7 +10,7 @@
     tools:context=".ui.dashboard.DashboardActivity">
 
     <android.support.design.widget.CollapsingToolbarLayout
-        android:layout_width="match_parent"
+        android:layout_width="match_parengit commi"
```

Comment:
Are you ok? Probably you started type in this file instead terminal. Hahaha Don't worry, it happen!